federation/pkg/typeadapters/adapter.go

```diff
@@ -14,31 +14,35 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package framework
+package typeadapters
```

Comment:
This change has moved to #42025.  Not sure which is preferable, there seems to be both plural and singular package names in the tree.